test/e2e/templates/done.tpl

```diff
@@ -2,27 +2,42 @@
 // OIDC Redirect Flow - this is the page that is redirected to with
 // tokens in the parameters
 
+// PKCE cannot be enabled because the test app is a "web" type. OKTA-246000 
+var pkce = false;
```

Comment:
Is this why we are setting `pkce: false` to a bunch of tests in the widget ?

Reply:
In the E2E tests doing OIDC flow, yes. They were built using implicit flow. I have a [task](https://oktainc.atlassian.net/browse/OKTA-246000) to add a SPA client ID to the test config. I would probably want to add this to the 4.0 branch before we merge to master.